talk/morelanguage/stl.tex

```diff
@@ -239,18 +258,44 @@
 \end{frame}
 
 \begin{frame}[fragile]
-  \frametitlecpp[14]{STL and lambdas}
+  \frametitlecpp[14]{More examples}
   \begin{cppcode}
-    // Finds the first element in a list between 1 and 10.
     std::list<int> l = ...;
-    ...
+
+    // Finds the first element in a list between 1 and 10.
     const auto it =
       std::find_if(l.begin(), l.end(),
         [](int i) { return i >= 1 && i <= 10; });
+    if (it != l.end())
+      int element = *it;
```

Comment:
This is a bit pointless, as `element` will immediately go out of scope.

Reply:
Fixed.